cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/common/DeterministicCluster.java

```diff
@@ -638,6 +647,961 @@ public static ClusterModel minLeaderReplicaPerBrokerSatisfiable3() {
+    cluster.setReplicaLoad(RACK_BY_BROKER4.get(3)
+                                   .toString(), 3, topic1Partition3, createLoad(1.0, 1.0, 1.0, 1.0), windows);
+
+    // T0_P0_follower_1
+    cluster.createReplica(RACK_BY_BROKER4.get(2)
+                                  .toString(), 2, topic0Partition0, 1, false);
+    cluster.setReplicaLoad(RACK_BY_BROKER4.get(2)
+                                   .toString(), 2, topic0Partition0, createLoad(1.0, 1.0, 1.0, 1.0), windows);
+    // T0_P1_follower_1
+    cluster.createReplica(RACK_BY_BROKER4.get(0)
+                                  .toString(), 0, topic0Partition1, 1, false);
+    cluster.setReplicaLoad(RACK_BY_BROKER4.get(0)
+                                   .toString(), 0, topic0Partition1, createLoad(1.0, 1.0, 1.0, 1.0), windows);
+    // T0_P2_follower_1
+    cluster.createReplica(RACK_BY_BROKER4.get(1)
+                                  .toString(), 1, topic0Partition2, 1, false);
+    cluster.setReplicaLoad(RACK_BY_BROKER4.get(1)
+                                   .toString(), 1, topic0Partition2, createLoad(1.0, 1.0, 1.0, 1.0), windows);
+    // T0_P3_follower_1
+    cluster.createReplica(RACK_BY_BROKER4.get(1)
+                                  .toString(), 1, topic0Partition3, 1, false);
+    cluster.setReplicaLoad(RACK_BY_BROKER4.get(1)
+                                   .toString(), 1, topic0Partition3, createLoad(1.0, 1.0, 1.0, 1.0), windows);
+    // T1_P0_follower_1
+    cluster.createReplica(RACK_BY_BROKER4.get(4)
+                                  .toString(), 4, topic1Partition0, 1, false);
+    cluster.setReplicaLoad(RACK_BY_BROKER4.get(4)
+                                   .toString(), 4, topic1Partition0, createLoad(1.0, 1.0, 1.0, 1.0), windows);
+    // T1_P1_follower_1
+    cluster.createReplica(RACK_BY_BROKER4.get(3)
+                                  .toString(), 3, topic1Partition1, 1, false);
+    cluster.setReplicaLoad(RACK_BY_BROKER4.get(3)
+                                   .toString(), 3, topic1Partition1, createLoad(1.0, 1.0, 1.0, 1.0), windows);
+    // T1_P2_follower_1
+    cluster.createReplica(RACK_BY_BROKER4.get(4)
+                                  .toString(), 4, topic1Partition2, 1, false);
+    cluster.setReplicaLoad(RACK_BY_BROKER4.get(4)
+                                   .toString(), 4, topic1Partition2, createLoad(1.0, 1.0, 1.0, 1.0), windows);
+    // T1_P3_follower_1
+    cluster.createReplica(RACK_BY_BROKER4.get(4)
+                                  .toString(), 4, topic1Partition3, 1, false);
+    cluster.setReplicaLoad(RACK_BY_BROKER4.get(4)
+                                   .toString(), 4, topic1Partition3, createLoad(1.0, 1.0, 1.0, 1.0), windows);
+
+    return cluster;
+  }
+
+  /**
+   * 6 brokers, 2 BrokerSets, each BrokerSet has 3 Brokers
+   * 2 topics, each topic has 2 partitions, each partition has 2 replicas.
+   * Topics are not contained within the boundary of a brokerset.
```

Comment:
Looks like the description is wrong here. T0 only resides on [B0,B1,B2] and T1 only resides on [B3,B4,B5], so topics ARE contained within the boundary of brokerset?

Reply:
Good catch. Updating.